app/serializers/v1/workflow/step_assignment_serializer.rb

```diff
@@ -0,0 +1,20 @@
+class V1::Workflow::StepAssignmentSerializer < ApplicationSerializer
+  set_id :created_at # exception to the rule, we don't care about sharing this
```

Comment:
Why are we setting the id to `created_at` timestamp? Shouldn't we just use the id?